Sources/ShopifyCheckoutKit/CheckoutWebView.swift

```diff
@@ -189,7 +207,7 @@ extension CheckoutWebView: WKNavigationDelegate {
 			ShopifyCheckoutKit.configuration.logger.log(message)
 			CheckoutBridge.instrument(self, InstrumentationPayload(name: "checkout_finished_loading", value: Int(diff * 1000), type: .histogram, tags: ["preloading": preloading]))
 		}
-
+		checkoutDidLoad = true
```

Comment:
Whereabouts is this reset to `false`?

Reply:
Each preload creates a new view, so this is thrown away with each preload.

The view is also discarded away whenever we clear the view cache (complete, cancel, error).